### DISCUSSION TITLE
Simultaneous video and ambience music playback.

### DISCUSSION THREAD

**Discussion by SquareCircle99** [sircharlo/meeting-media-manager#858] [Ideas] [answered]
Hello all! In our meetings, we play a 5 minute countdown video to let everyone know that the meeting's about to start and that they should look for their seats. Up until two or three releases ago, the ambience music would continue to play while the video played at the same time. This is no longer the case, as the music stops when we click play on the video. I understand the reasoning behind this, but could perhaps a toggle be added to the settings to allow for simultaneous playback of media and ambience music?
Thank you!

**Comment by mtdvlpr** [comment, marked as answer]
Yes of course! Sorry this happened. I wasn't aware of such a scenario.

**Reply by SquareCircle99** [reply]
Thank you! Much appreciated.

**Comment by mtdvlpr** [comment]
It's fixed in d1cc4a772f86a344d6cb298929743716a1b08d52. I just removed this piece of automation altogether, since I think it's too small to make it a feature to be toggled.

**Comment by mtdvlpr** [comment]
Hi @SquareCircle99, while you're waiting for an official new release, [here](https://drive.google.com/drive/folders/1_2CzKieYGaK3LcYrEwzNQgfTpNReOSx9?usp=share_link)'s a version which just fixes your issue :)

**Reply by SquareCircle99** [reply]
Thank you. I shall install it tomorrow!
